src/handle-initial-jsmd.js

```diff
@@ -27,4 +27,27 @@ export default async function handleInitialJsmd(store) {
       store.dispatch(evaluateAllCells())
     }
   }
+
+  state = store.getState();
```

Comment:
This doesn't really belong here, but I couldn't find another place to do sanity checks right after loading a notebook.  Any suggestions?

Reply:
(although now that I look at it, I don't think you should be using redux for this particular use case)